app/javascript/lib/imperial.test.js

```diff
@@ -1,8 +1,14 @@
 import Action from './action';
+
+import Logger from '../src/Logger';
+
 import { Bond, Nation } from './constants';
 import GameBoard from './gameBoard';
+
 import Imperial from './imperial';
 
+const newImperialGame = (board) => new Imperial(board, new Logger('unittests', 'stub-test-id'));
```

Comment:
Same q, can we remove the Logger?
Hmm and without the logger I'm not sure what benefit this refactor gets us.

Reply:
[76f61f0](https://github.com/Thrillberg/imperial/pull/729/commits/76f61f00f58e876e15be9e5e2d7e0ddfdbd8a432)

Reply:
Oh, there's also a tiny refactor where I removed `static Imperial.fromLogs` and made it a class method `new Imperial().tickFromLog()`

Reply:
Yeah I think that refactor is good but changing `new Imperial(board)` to `newImperialGame(board)` doesn't make a lot of sense to me, if `newImperialGame()` isn't bringing any other logic in or anything. (This is currently the case in `imperial.test.js`)

Reply:
Oh no wait, I was mistaken, thought you're referring to `newImperialFromLog`, yeah, in that case now that we've removed the logger this is unnecessary

Reverting

Reply:
[7cdd568](https://github.com/Thrillberg/imperial/pull/729/commits/7cdd568bd4c56ec463bf0db35fd5482eb7dfe399)